### PR TITLE
Adding backup mainnet seeds to mainnet.txt

### PR DESCRIPTION
**Pull request by StevenMartinez94**
These are some nodefleet's mainnet archival nodes that people can use as seeds in case default mainnet seeds list fails.

### CODE REVIEW

File: mainnet.txt [hunk 1]
```diff
@@ -3,3 +3,8 @@ cdcf936d70726dd724e0e6a8353d8e5ba5abdd20@seed2.mainnet.pokt.network:26663
 74b4322a91c4a7f3e774648d0730c1e610494691@seed3.mainnet.pokt.network:26662
 b3235089ff302c9615ba661e13e601d9d6265b15@seed4.mainnet.pokt.network:26663
```
Comment (msmania) [inline mainnet.txt]: Is it ok to list six nodes equally so that the script in README.md works?

**Review by msmania — CHANGES_REQUESTED** [review]

File: mainnet.txt (hunk 1 above)
Comment (aqt01) [reply mainnet.txt]: @msmania sure! our socket inlcudes 4 pocket nodes but we can add +2 to match then. we'll coordinate this with PNF!

Comment (StevenMartinez94) [reply mainnet.txt]: > Is it ok to list six nodes equally so that the script in README.md works?

Done

File: mainnet.txt [hunk 2]
```diff
@@ -2,4 +2,11 @@
 cdcf936d70726dd724e0e6a8353d8e5ba5abdd20@seed2.mainnet.pokt.network:26663
 74b4322a91c4a7f3e774648d0730c1e610494691@seed3.mainnet.pokt.network:26662
 b3235089ff302c9615ba661e13e601d9d6265b15@seed4.mainnet.pokt.network:26663
+ab5776d1c9a50d47e30e90a0600e787d9fb74643@seed5.mainnet.pokt.network:26664
+a6dec84acfe1bae887cd6c4f2eeb864fae29c9f7@seed6.mainnet.pokt.network:26664
 
+Also you can use these seeds if the list above fails for any reason:
```
Comment (msmania) [inline mainnet.txt]: Can you remove these lines 8-12?  They're duplicated.

**Review by msmania — CHANGES_REQUESTED** [review]

File: mainnet.txt (hunk 2 above)
Comment (StevenMartinez94) [reply mainnet.txt]: Done

**Review by msmania — APPROVED** [review]
Thanks!

**Pull request merged**